### ISSUE TITLE
Above bridge speed adjustment

### ISSUE THREAD

**Issue by Ryan6338** [supermerill/SuperSlicer#2902]
When using low layer heights, the layer above bridges needs to be printed much slower than regular infill to achieve good adhesion as the thin bridge layer just warps under the nozzle pressure rather than fusing. It would be great to be able to independently adjust above bridge speed so the whole print speed does not need to be slowed to overcome this issue.

**Comment by SimonSolar2C** [comment]
Yes this can be done tediously but would be so nice as a tick box.  even a double bridge layer would work

**Comment by SimonSolar2C** [comment]
I really need this now because I do nice bridge layers then the next layer catches it and clumps up and make a hole in my nice bridging.
